CODEOWNERS

```diff
@@ -91,6 +91,9 @@ Kconfig*                                  @tejlmand
 /samples/peripheral/radio_test/README.rst @b-gent
 /samples/usb/usb_uart_bridge/             @jhn-nordic @jtguggedal
 /samples/CMakeLists.txt                   @tejlmand
+/samples/openthread/coap_client/          @lmaciejonczyk
```

Comment:
I think you could handle the codeowners just like @MarekPorwisz did in his PR:
```
/samples/openthread/                      @MarekPorwisz @lmaciejonczyk
/samples/openthread/**/README.rst         @greg-fer
```
It's more concise, and we'll avoid redundant entries after both PRs are merged.

Reply:
Changed